### PR TITLE
Enable PHP 7.4-8.1 comat

### PR DESCRIPTION
**Pull request by koriym**
* Run PHP 7.4-8.1 tests 
* Migrate CI from Travis to GH action
* Update CI badges (scrutinizer, codecov and GH action)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,25 +20,15 @@
...
-            "config": {
-                "common": "Aura\\Accept\\_Config\\Common"
-            }
+            "Aura\\Accept\\": "src/"
         }
     },
     "require-dev": {
-        "aura/di": "~2.0",
-        "phpunit/phpunit": "~5.7 || ~4.8"
+        "yoast/phpunit-polyfills": "^1.0"
```
Comment (harikt) [inline composer.json]: Do we need polyfill

**Review by harikt — COMMENTED** [review]
May be push to 7.4 ?

**Comment by koriym** [comment]
I thought about it again.

Aura.Session droped support for lower versions of PHP, with good results, but we don't have that advantage here, and it's not likely to happen any time soon. We also know that every few years phpunit breaks its backwards compatibility.

Shouldn't we move to 4.x when there is an obvious disadvantage to supporting older PHP versions or an obvious advantage to dropping support for older PHP? It seems to me that it would be more stable to use `phpunit-polyfills` with 3.x as it is done with other Aura packages.

**Comment by harikt** [comment]
If they are using 5.x version of php 2.x will continue to work. And now if we look at the current situation 7.4 is at security only release mode. So supporting below 7.4 may be more work when php moves forward with more breaks. ie why I was suggesting going with 7.4 +

**Comment by harikt** [comment]
The tests is failing due to travis not working. I am gng to merge this as is. In case of failures please do check.

**Comment by koriym** [comment]
@harikt Ok thanks.

**Pull request merged**